### ISSUE TITLE
After changing permissions I need to reconnect

### ISSUE THREAD

**Issue by MartinX3** [masneyb/gftp#147]
I can change the permission of a file once.
On the second try I need to reconnect, because of the error
```
SITE CHMOD 705 Forsbach-Tour 78km 2022.gpx
200 Permissions changed on Forsbach-Tour 78km 2022.gpx
Ordnerliste /www/downloads wird von der Gegenstelle geladen (LC_TIME=de_DE.UTF-8)
EPSV
229 Extended Passive mode OK (|||25088|)
MLSD
150 Accepted data connection
226-Options: -a -l 
226 52 matches total
The server doesn't support SITE CHMOD
```

**Comment by wdlkmpx** [comment]
The issue happens after commit https://github.com/masneyb/gftp/commit/9867d7f34b84ce23f8923defd993f9c97650f05e

It's missing `== 500` for some reason

**Comment by wdlkmpx** [comment]
I should have released v2.9.2b with this fix, the bug produces a lot of noise

**Comment by MartinX3** [comment]
xD

The slow transfer speed is worse :)
